audmetric/core/api.py

```diff
@@ -179,6 +182,107 @@ def confusion_matrix(
     return matrix
 
 
+def detection_error_tradeoff(
+    truth: typing.Union[
+        typing.Union[int, float],
+        typing.Sequence[typing.Union[int, float]]
+    ],
+    prediction: typing.Union[
+        typing.Union[int, float],
+        typing.Sequence[typing.Union[int, float]]
+    ],
+) -> typing.Tuple[np.ndarray, np.ndarray, np.ndarray]:
+    r"""Detection error tradeoff for verification experiments.
+
+    The `detection error tradeoff (DET)`_
+    is a graph showing
+    the false non-match rate (FNMR)
+    against the false match rate (FMR).
+    The FNMR indicates
+    how often an enrolled speaker was missed.
+    The FMR indicates
+    how often an impostor was verified as the enrolled speaker.
+
+    This function does not return a figure,
+    but the FMR and FNMR,
+    together with the corresponding verification thresholds
+    at which a similarity value
+    was regarded to belong to the enrolled speaker.
+
+    ``truth`` may only contain entries like ``[1, 0, True, False...]``,
+    whereas prediction values
+    can also contain similarity scores, e.g. ``[0.8, 0.1, ...]``.
+
+    The implementation is identical with the one provided
+    by the pyeer_ package.
+
+    .. _detection error tradeoff (DET): https://en.wikipedia.org/wiki/Detection_error_tradeoff
+    .. _pyeer: https://github.com/manuelaguadomtz/pyeer
+
+    Args:
+        truth: ground truth classes
+        prediction: predicted classes
```

Comment:
Should we add "or similarity values"? I know, it's written in the description above.

Reply:
No, good idea, I added it as this is also the more likely scenario how to use it.